crates/accelerate/src/circuit_library/mod.rs

```diff
@@ -13,12 +13,15 @@
 use pyo3::prelude::*;
 
 mod entanglement;
+mod iqp;
 mod pauli_feature_map;
 mod quantum_volume;
 
 pub fn circuit_library(m: &Bound<PyModule>) -> PyResult<()> {
     m.add_wrapped(wrap_pyfunction!(pauli_feature_map::pauli_feature_map))?;
     m.add_wrapped(wrap_pyfunction!(entanglement::get_entangler_map))?;
+    m.add_wrapped(wrap_pyfunction!(iqp::py_iqp))?;
+    m.add_wrapped(wrap_pyfunction!(iqp::py_random_iqp))?;
```

Comment:
I don't think it matters too much, but this uses a slightly different naming convention from most of the other circuit library code (i.e. here the functions are prefixed with `py`).